tests/grpc/call_test.go

```diff
@@ -115,7 +116,11 @@ func TestCall(t *testing.T) {
 				"r1": property.New(property.Computed).WithDependencies([]resource.URN{
 					"urn1", "urn2",
 				}),
+				"r2": property.New("s"),
 			}),
+			ReturnDependencies: map[string][]resource.URN{
+				"r2": {resource.URN("urn1"), resource.URN("urn2")},
+			},
```

Comment:
Here we add another return property `r2` that is encoded in an old style, e.g. as may be returned from a wrapped legacy provider. Of course, `r1` is in the new style.